nodelet_plugins.xml

```diff
@@ -1,5 +1,9 @@
 <library path="lib/libopencv_apps">
 
+  <class name="opencv_apps/adding_image" type="adding_image::AddingImageNodelet" base_class_type="nodelet::Nodelet">
```

Comment:
please follow old classname convention, this should be "adding_image/adding_image" until https://github.com/ros-perception/opencv_apps/pull/21